third_party/terraform/tests/resource_compute_instance_template_test.go

```diff
@@ -476,21 +476,15 @@ func TestAccComputeInstanceTemplate_EncryptKMS(t *testing.T) {
 	t.Parallel()
 
 	var instanceTemplate compute.InstanceTemplate
-
-	org := getTestOrgFromEnv(t)
-	pid := "tf-test-" + acctest.RandString(10)
-	billingAccount := getTestBillingAccountFromEnv(t)
-	diskName := fmt.Sprintf("tf-test-%s", acctest.RandString(10))
-	keyRingName := fmt.Sprintf("tf-test-%s", acctest.RandString(10))
-	keyName := fmt.Sprintf("tf-test-%s", acctest.RandString(10))
+	kms := bootstrapKMSKey(t)
```

Comment:
Do you think [near-]future tests will need anything but the Crypto Key's name? We could shrink the "public" surface a little by only providing that.

Reply:
Trying to add my second test I've run into needing a reference to the key ring :(